### PR TITLE
setup/customization: add acpid example

### PR DESCRIPTION
**Pull request by tormath1**
Signed-off-by: Mathieu Tortuyaux <mtortuyaux@microsoft.com>

<hr>

Related to:
- https://github.com/flatcar-linux/coreos-overlay/pull/1911
- https://github.com/flatcar-linux/Flatcar/issues/738

### CODE REVIEW

File: docs/setup/customization/ACPI.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: Handle ACPI events
+linktitle: ACPI
+description: Enable ACPId and handle ACPI events
+weight: 60
+---
+
+## ACPId
```
Comment (krnowak) [inline docs/setup/customization/ACPI.md:8]: Not sure if I have ever seen this way of writing "acpid".

```suggestion
## acpid
```

File: docs/setup/customization/ACPI.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+Beginning with Flatcar major release XXXX `acpid` can be enabled at boot with Ignition.
```
Comment (krnowak) [inline docs/setup/customization/ACPI.md:10]: So, does this PR need to wait until we know what will be the next release number?

File: docs/setup/customization/ACPI.md (hunk 1 above)
Comment (tormath1) [reply docs/setup/customization/ACPI.md:8]: Yeah - I took some artistic freedom. Let's stuck to `acpid`.

**Review by pothos — COMMENTED** [review]
For consistency it would be good to link to this document from `docs/_index.md` as done with other pages.

**Comment by tormath1** [comment]
@pothos thanks for the review - comment has been addressed :)

File: docs/setup/customization/ACPI.md (hunk 2 above)
Comment (pothos) [reply docs/setup/customization/ACPI.md]: It's not in the flatcar-3248 coreos-overlay branch, and needs a cherry-pick and rebuild of the started Alpha. So I think it's maybe ok to say > 3248 or make up a number like 3250

Comment (tormath1) [reply docs/setup/customization/ACPI.md]: It's now cherry-picked to flatcar-3248 :)

**Review by krnowak — APPROVED** [review]

**Pull request merged**